element_calcium_imaging/scan.py

```diff
@@ -443,41 +444,40 @@ def estimate_nd2_scan_duration(nd2_scan_obj):
                 )
         elif acq_software == "PrairieView":
             from element_interface import prairieviewreader
-            
 
             pvscan_filepaths = get_prairieview_files(key)
             pvscan_info = prairieviewreader.get_pv_metadata(pvscan_filepaths[0])
-
             self.insert1(
                 dict(
                     key,
-                    nfields = pvscan_info["num_fields"],
-                    nchannels = pvscan_info["num_nchannels"],
-                    ndepths = pvscan_info["num_planes"],
-                    nframes = pvscan_info["num_frames"],
-                    rois = pvscan_info["num_rois"],
-                    x = pvscan_info["x_pos"],
-                    y = pvscan_info["y_pos"],
-                    z = pvscan_info["z_pos"],
-                    fps = pvscan_info["frame_rate"],
-                    usecs_per_line = pvscan_info["usecs_per_line"],
-                    scan_datetime = pvscan_info["scan_datetime"],
-                    scan_duration = pvscan_info["scan_duration"],
+                    nfields=pvscan_info["num_fields"],
+                    nchannels=pvscan_info["num_nchannels"],
+                    ndepths=pvscan_info["num_planes"],
+                    nframes=pvscan_info["num_frames"],
+                    rois=pvscan_info["num_rois"],
+                    x=pvscan_info["x_pos"],
+                    y=pvscan_info["y_pos"],
+                    z=pvscan_info["z_pos"],
+                    fps=pvscan_info["frame_rate"],
+                    usecs_per_line=pvscan_info["usecs_per_line"],
+                    scan_datetime=pvscan_info["scan_datetime"],
+                    scan_duration=pvscan_info["scan_duration"],
                 )
-            
             )
 
-            
             self.Field.insert(
                 dict(
                     key,
-                    field_idx = plane_idx,
-                    px_height = pvscan_info["height_in_pixels"],
-                    px_width = pvscan_info["width_in_pixels"],
-                    um_height = pvscan_info["height_in_um"],
-                    um_width = pvscan_info["width_in_um"],
-                    )
-                    for plane_idx in range(pvscan_info["num_planes"])
+                    field_idx=plane_idx,
```

Comment:
This seems a little strange that the same pvscan information is populated for all planes

Reply:
The x and y coordinates do not seem to change over time. This system is only acquiring in different z-planes over time.

Reply:
@kushalbakshi Perhaps the example file does not represent all cases.  Is it possible to extract the x position, y position, and dimensions of the fields at each depth?

Reply:
The only x and y coordinates present in the metadata file are for the initial position when the scan is started and these coordinates are the center of the scanning field. @kabilar and I came to the conclusion that this scanner likely scans the same x and y coordinates for the entire scan. In the z-plane, it scans through a user-defined number of planes (ndepths). Hopefully this clarifies it a bit further, @dimitri-yatsenko

Reply:
Thanks for checking @kushalbakshi.  I will go ahead and merge.